docs/pages/components/loading-spinner.md

```diff
@@ -30,24 +41,17 @@ The loading element is used to display the loading indicator animation.
 
 <br>
 
-## Usage with other elements
-The loading indicator is most often used within another component to indicate the loading state. A container (even `ui-`, `app-`, and page-level) can be “activated” by setting `aria-busy="true"`.
-
-The spinner should be included inside the container. Visibility can be toggled in relation to the aria-busy attribute. They should always be opposites, i.e, if currently loading, `section[aria-busy="true"]`, `.fd-spinner[aria-hidden="false"]`, once the content is loaded, toggle to false and true respectively.
-
-{% capture aria %}
-<div class="fd-panel" aria-busy="true">
-    <div class="fd-spinner" aria-hidden="false" aria-label="Loading">
-        <div></div>
-    </div>
-    <div class="fd-panel__header">
-        <h3 class="fd-panel__title">Lorem ipsum</h3>
-    </div>
-    <!-- Loaded content goes here -->
-    <div class="fd-panel__footer">
-        <p><em>Etiam pulvinar turpis sed velit porttitor vel adipiscing velit fringilla.</em></p>
-    </div>
+
+
+## Spinner (deprecated)
```

Comment:
We should just remove this. We're about to release a breaking change so might as well.